content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -176,6 +176,11 @@ The following methods exist for installing kubectl on Linux:
 {{< note >}}
 In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default.
 You can create this directory if you need to, making it world-readable but writeable only by admins.
+
+```shell
+sudo mkdir -m 755 /etc/apt/keyrings
```

Comment:
Part of me thinks that if you need to be told this, you're in the wrong job.
However, I am also willing to see this go in.

I'd like to see what other reviewers think. Do we need to explain how `mkdir` works? The command is older than I am.

Reply:
I'm here to install a tool, not do a school test. It's obtuse to give a hint to the answer and to not just include it.

Reply:
More concise:


In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default, and can be created using `sudo mkdir -m 755 /etc/apt/keyrings`

If you expect competence with Linux file system permissions, then this would convey meaning in fewer characters

Reply:
Sure, try that.

Reply:
updated